src/modules/python.rs

```diff
@@ -365,6 +365,33 @@ prompt = 'foo'
         dir.close()
     }
 
+    #[test]
+    fn with_active_venv_and_dirty_prompt() -> io::Result<()> {
+        let dir = tempfile::tempdir()?;
+        create_dir_all(dir.path().join("my_venv"))?;
+        let mut venv_cfg = File::create(dir.path().join("my_venv").join("pyvenv.cfg"))?;
+        venv_cfg.write_all(
+            br#"
+home = something
+prompt = '(foo) '
```

Comment:
Is whitespace after right paren intentional?

Reply:
I guess not :upside_down_face: 
```suggestion
prompt = '(foo)'
```

Reply:
> I guess not 🙃

Oops! Good catch, that was unintentional. pipenv adds a trailing space to the prompt and I wanted make sure it was handled properly. Forgot to remove it 😕